macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -325,11 +325,31 @@ open class AvatarView: NSView {
 	///
 	/// - note: Internal visibility exists only for unit testing
 	@objc(getColorForIndex:)
+	@available(*, deprecated, message: "Use getInitialsColorSetFromPrimaryText:secondaryText: instead")
 	public static func getColor(for index: Int) -> ColorSet {
 		let avatarBackgroundColors = AvatarView.avatarColors
 		return avatarBackgroundColors[index % avatarBackgroundColors.count]
 	}
 
+	/// Returns a color table index for a given display name
+	/// - parameter identifyingString: the unique identifying string for the user
+	///
+	/// - Returns: a unique index generated by walking backwards through the letters in the \p identifyingString
```

Comment:
Is it unique? If it's a hash being generated from the string, uniqueness is not guaranteed.

Reply:
I don't know for sure, I haven't modified the implementation of original method.

Reply:
Let's remove "unique" from this comment. It may be misleading.

Reply:
I have removed unique from the description.